keyboards/kmac/readme.md

```diff
@@ -1,44 +1,21 @@
-# KMAC keyboard firmware
+# KMAC
 
 A Korean custom keyboard designed by Byungho Kim and the KBDMania community.
 
-## Supported models
+Keyboard Maintainer: [Mathias Andersson](https://github.com/wraul)
```

Comment:
```suggestion
Keyboard Maintainer: [Mathias Andersson](https://github.com/wraul)  
```

Reply:
I always forget that markdown don't preserve newlines.
It should be fixed now.